ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala

```diff
@@ -183,7 +183,7 @@ object Cli {
                 .fromCrtFile(path)
                 .valueOr(err => sys.error(s"Failed to create RSA256 verifier: $err"))))))
 
-      opt[String]("auth-jwt-ec256-crt")
+      opt[String]("auth-jwt-es256-crt")
```

Comment:
Hmmm… we _are_ changing the CLI switches, so I do think a changelog entry is in order now. People might be using them.